.github/workflows/engine-nightly.yml

```diff
@@ -19,6 +19,7 @@ jobs:
     steps: [
       { name: 'Checkout', uses: actions/checkout@v3 },
       { name: 'Install Python', uses: actions/setup-python@v3, with: { python-version: 3.x, architecture: x64 } },
+      { name: 'XCode', uses: maxim-lobanov/setup-xcode@v1, with: { xcode-version: '14.2' } },
```

Comment:
Is there no "official" setup action?

Reply:
Ah, I see we've used it before as well. Ok!

Reply:
DIdn't find one last time 🤷